main.tf

```diff
@@ -10,10 +10,12 @@ resource "aws_rds_cluster" "main" {
   master_password              = "${var.password}"
   port                         = "${var.port}"
   engine                       = "${var.engine}"
+  engine_version               = "${var.engine_version}"
   backup_retention_period      = 7
   preferred_backup_window      = "02:00-03:00"
   preferred_maintenance_window = "wed:04:00-wed:04:30"
   snapshot_identifier          = "${var.snapshot_identifier}"
+  final_snapshot_identifier    = "${var.name_prefix}-cluster"
```

Comment:
I was thinking this should only be set if `skip_final_snapshot` is `false` but it looks safe to always set it. I do think the name should be something like `"${var.name_prefix}-cluster-final-snapshot"`

Reply:
I'm worried that the name might be too long if we add "final-snapshot" in there :/

Reply:
Maybe just add final then? Would be nice to have something to easily see that its a final snapshot.